docs/source/notes/get_start_xpu.rst

```diff
@@ -55,17 +55,17 @@ Platform Windows
 
 Now we have all the required packages installed and environment activated. Use the following commands to install ``pytorch`` on Windows, build from source for ``torchvision`` and ``torchaudio``.
 
-For release wheels
+For preview wheels
 
 .. code-block::
 
-    pip3 install torch --index-url https://download.pytorch.org/whl/xpu
+    pip3 install torch --index-url https://download.pytorch.org/whl/test/xpu
 
 For nightly wheels
 
 .. code-block::
 
-    pip3 install --pre torch --index-url https://download.pytorch.org/whl/nightly/xpu
+    pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu
```

Comment:
Do you really want torchaudio here?

Reply:
I believe we do have both torchaudio and torchvision xpu builds: https://hud2.pytorch.org/hud/pytorch/audio/nightly/1?per_page=50&name_filter=xpu